deis-dev/manifests/deis-minio-rc.yaml

```diff
@@ -18,8 +18,24 @@ spec:
         - name: deis-minio
           image: quay.io/deisci/minio:v2-beta
           imagePullPolicy: Always
+          env:
+            - name: HEALTH_SERVER_PORT
+              value: "8082"
           ports:
             - containerPort: 9000
+            - containerPort: 8082
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8092
+            initialDelaySeconds: 30
+            timeoutSeconds: 1
```

Comment:
Should this default to 5 seconds, or do we intend to make that a pervasive change later.? See deis/workflow#344.